Makefile

```diff
@@ -2,9 +2,12 @@ fmt:
 	gofmt -s -w .
 
 fmtcheck:
-	(! gofmt -s -d . | grep '^')
```

Comment:
Current version of `gofmt` outputs stderr instead of stdout if there's some failure.

Reply:
I want to get non-zero exit status when format check fails. That's why I wrote the task like this.

Reply:
ahh, that's my fault, I was running gofmt to files which has invalid syntax. (In this case gofmt returns non-zero exit status)
I've rollbacked it:)